add-domain

```diff
@@ -999,3 +999,32 @@ wiflix.xyz
 wiflix.yachts
 coronavirus.zone
 fullcopec.zone
+dexsceerner.net
+app.dexscreener-home.net
+dexscreener-home.net
+traolerjoe.org
+traderjoexyz.bylfg.org
+bylfg.org
+v2-o-p-e-n-s-e-a.com
+susni-swap.com
+sushi.swap-ether.site
+swap-ether.site
+open-sea.market-ntf.com
+market-ntf.com
+hyperilquid.org
+hyperilquid.xyz-trade.com
+xyz-trade.com
+tangem.ing
+tacngcm.com
+raydiumx.org
+ray-swap.net
+raydium.io-sol.org
```

Comment:
Double of `io-sol.org`

- https://kb.mypdns.org/issue/MTX-127025/io-sol.org MTX-127025
- https://mypdns.youtrack.cloud/issue/MTXB-209543/io-sol.org MTXB-209543
- https://github.com/mypdns/matrix/issues/127025